### ISSUE TITLE
New label does not appear after selected.

### ISSUE THREAD

**Issue by rishav4101** [bench-routes/bench-routes#436]
**Describe the bug**
In the quick input page, when a new custom label is created and then selected from dropdown, it ceases to appear in the dropdown. So once a custom label is selected it cannot be unselected.

**To Reproduce**
Steps to reproduce the behaviour:
1. Go to Quick Input page.
2. Click on Add labels.
3. Add a custom new label and select it.

**Error screenshot**
![label1](https://user-images.githubusercontent.com/55195287/112710087-9812f900-8ee4-11eb-922e-51f94e27beb1.JPG)

**Expected behaviour screenshot**
![label2](https://user-images.githubusercontent.com/55195287/112710127-e1fbdf00-8ee4-11eb-842a-34afbc450275.JPG)

**Desktop :**
 - OS: ubuntu

**Comment by rishav4101** [comment]
@muskankhedia @aquibbaig I would like to work on this issue.

**Comment by aquibbaig** [comment]
https://github.com/bench-routes/bench-routes/pull/380#pullrequestreview-622637797

**Comment by rishav4101** [comment]
@aquibbaig Yes, and that is the reason the newly created label does not appear in the dropdown once selected.

**Comment by aquibbaig** [comment]
In that case, let's wait for the PR.

**Comment by DivyanshiSingh** [comment]
If it is not assigned to anyone else, I would like to work on it @aquibbaig

**Comment by aquibbaig** [comment]
This issue depends on PR #380. I'd have to ask @codetheorem if he's currently working on a fix.

**Comment by DivyanshiSingh** [comment]
Sure, please do let me know.

**Comment by codetheorem** [comment]
I have already solved this issue in my PR #380

**Comment by RiyaGupta89** [comment]
Is this issue fixed or is still on. If it is on, then may I work on this issue?

**Comment by muskankhedia** [comment]
@RiyaGupta89 The dashboard v1.1 has been deprecated now and not maintained actively. A new version of UI, more modular and compact, for bench-routes is in active development [here](https://github.com/bench-routes/dashboard). You can go through the issues and also report for new features/bugs in the dashboard repo to get started with the project
